src/main/java/org/sonar/squidbridge/annotations/AnnotationBasedRulesDefinition.java

```diff
@@ -43,6 +43,13 @@
 import org.sonar.squidbridge.rules.ExternalDescriptionLoader;
 
 /**
+ * @deprecated since 2.7 without alternative.
+ *
+ * Responsibility of loading rule metadata into RulesDefinition is let to custom plugins.
+ * Most of the SQALE annotations are ignored, and not supported.
```

Comment:
What does "not supported" means here?
Did we change something in the code of sslr-squid-bridge to ignore/stop supporting anything?

Reply:
The current method call, such as `setDebtSubCharacteristic` won't have any effect anymore. I plan to even remove it in PR updating to 6.7.